### DISCUSSION TITLE
Should we translate source code outputs and comments?

### DISCUSSION THREAD

**Discussion by GasimGasimzada** [reactjs/reactjs.org-translation#234] [Team Posts]
I understand the translating variables names is not a good idea and all. However, I would like to know if we should translate outputs (For example, should we translate "Hello Taylor" in home page examples)? Also, what about comments? Should we translate them or keep them in English? 

My main point is, should sample source codes stay the same across all translations?

**Comment by simsim0709** [comment]
That's a good point. As a reference, Vue translated comments and string value(ex: Hello Taylor).
IMO, we'd better do it as well.

**Comment by GasimGasimzada** [comment]
That's what we ended up doing. All outputs (labels, texts, placeholders etc) are translated and localized (e.g currency changed to local currency, names changed to local names). Comments are also translated (some examples have Notes in them).
